dev/run-tests-jenkins.py

```diff
@@ -21,9 +21,17 @@
 import os
 import sys
 import json
-import urllib2
 import functools
 import subprocess
+if sys.version < '3':
```

Comment:
@WeichenXu123 .
Given that the shebang at line 1 is `#!/usr/bin/env python3`, when this `if` become `true`?